test/helpers/pxpotrf.fypp

```diff
@@ -140,7 +140,6 @@ contains
                if (.not. allclose(R_dlaf, R_scalapack, uplo='L')) then
                   failed = .true.
                   write (error_unit, *) "ERROR: DLAF != ScaLAPACK"
-                  call terminate(ictxt)
```

Comment:
Can you explain this change? Was this a temporary change or is it meant to stay? I guess it was there for the buggy 2.2.2 release, but could maybe be put back with the pinned version?

Reply:
This is a leftover, I'll remove it.

Reply:
Actually, sorry, I misread the change (for some reason I read the change as if this was _added_). This was removed because checks are performed only by the main rank and `failed` is then broadcasted.